src/Http/Http.Results/test/ResultsOfTTests.Generated.cs

```diff
@@ -72,7 +72,7 @@ Results<ChecksumResult1, NoContent> MyApi()
         // Act & Assert
         var result = MyApi();
 
-        Assert.ThrowsAsync<ArgumentNullException>(async () =>
+        await Assert.ThrowsAsync<ArgumentNullException>(async () =>
```

Comment:
I'm kind of surprised there isn't an analyzer for this.

Reply:
I think there is now, which is why we noticed this.
See https://github.com/dotnet/aspnetcore/pull/57303